src/dotnet/Users.Service/Db/DbUserConverter.cs

```diff
@@ -0,0 +1,22 @@
+using Stl.Fusion.EntityFramework.Authentication;
+
+namespace ActualChat.Users.Db;
+
+public class DbUserConverter : DbUserConverter<UsersDbContext, DbUser, string>
+{
+    public DbUserConverter(IServiceProvider services) : base(services)
+    { }
+
+    public override void UpdateEntity(User source, DbUser target)
+    {
+        base.UpdateEntity(source, target);
+
+        // TODO: map unconditionally when user always comes filled in
```

Comment:
Yep, I would always set the `Status` field value here.

Reply:
Missing claim should mean the `Status` is default (0 = `Active`, I guess).

Reply:
this code has gone